src/main/java/uk/ac/ebi/pride/gui/util/UpdateChecker.java

```diff
@@ -94,12 +94,13 @@ private boolean isHigherVersion(String currentVersion, String version) {
      */
     public static void showUpdateDialog() {
         int option = JOptionPane.showConfirmDialog(null, "<html><b>A new version of ProteomeXchange submission tool is available</b>.<br><br> " +
-                "Would you like to update?</html>", "Update Info", JOptionPane.YES_NO_OPTION);
+                "Please download latest version by clicking yes </html>", "Update Info", JOptionPane.YES_NO_OPTION);
 
         if (option == JOptionPane.YES_OPTION) {
             DesktopContext context = Desktop.getInstance().getDesktopContext();
             String website = context.getProperty("px.submission.tool.url");
             HttpUtil.openURL(website);
         }
+        System.exit(0);
```

Comment:
instead of forcing to download latest version for every release ..it's good to do it based on some flag in API response..

Reply:
or may be based versioning criteria like if current version is x.y.z & latest version is x.y+1.z them may be we can force.. no need to force for x.y.z+1 etc.,